maths-club-pack/content/facilitator/monkey-business.md

```diff
@@ -0,0 +1,50 @@
+# Monkey Business
+
+## Introduction
+
+This is another problem where the solution seems impossible because there are too many things to consider (1000 monkeys!), but again becomes achievable once simplified and able to see patterns. To start the problem you should ensure students understand the rules, perhaps imitating with 4 students and the same number of coins or cards that can be flipped to face up or down.   
+
+The first student flip all cards up (u):&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[u, u, u, u]   
```

Comment:
This might be easier to format as a table than using spaces, but it is still fine like this